### DISCUSSION TITLE
Empty SubMenus are Shown

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#11566] [General] [answered]
Hi,
With the recent theia release v1.28.0, empty submenu's are displayed. Earlier these submenus were hidden as it didn't have any visible`MenuAction` in it. i.e MenuActions are registered to the submenu's path but are hidden with isVisible property. 

Is this an issue or is this expected ?

Thanks

**Comment by vince-fugnitto** [comment, marked as answer]
@maheshrajrp thank you for the discussion, it looks like a bug to me :+1: I'll create an issue shortly but will try a `git bisect` first.
